src/apps/bristleback_apps/bristleback_apps_common/memfault_platform_config.h

```diff
@@ -18,4 +18,4 @@
 // Send memfault metric updates every six hours
 #define MEMFAULT_METRICS_HEARTBEAT_INTERVAL_SECS (6 * 60 * 60)
 
-#define MEMFAULT_WATCHDOG_SW_TIMEOUT_SECS 30
+#define MEMFAULT_WATCHDOG_SW_TIMEOUT_SECS 4
```

Comment:
Not used, but if the software watchdog is ever enabled again in the future, I set it to 4 since our hardware watchdog is I believe 5 seconds.